st-embroider/2023/2003-01-23.md

```diff
@@ -0,0 +1,130 @@
+    - the assumption is that there are "classic ways" that the assets are getting included, if ember-auto-import also dealt with them there would be duplicates 
+  - there should be a way (config) to include your app in the ember-auto-import work
+    - e.g. you could have a particular folder that is just "normal js"
+  - one problem of classic app format is that `app/` is the entry point for everythign so you can't import anything outside it
+  - 22a: should we just upgrade our addons to v2 or do we need to invent this new config?
+  - ef4: yes probably, but this is still useful
+  - the actual config option would be to put a subdirectory of app under the remit of ember-auto-import
+  - not a super hard implementation, but we would need to make sure that the classic build would not see it
+    - use broccoli funnel to remove it
+  - it should be straightforward if we force people to be explicit in their config
+  - it all gets handled in the same way that 3rd party stuff would get handled
+  - you would probably need to rename relative imports to be the fully resolved `your-app-name/thing`
+    - i.e. ember-auto-import would do this automatically
+  - this could even allow you to progressivly move to embroider one folder at a time from a classic build
+
+
+<5 min> (?) recommended approach for inline svgs <@22a>
+  - similar converation to the last one
+  - CM: there are conversation happening in addons e.g. ember responsive image that help us think about things differently
+  - CM: I think there is a conversation in the svg jar addon but I'm not sure
+  - EF: you could convert all SVGs to components, but you could also implement a simple loader for svgs
+
+<15 min> discuss next steps of https://github.com/embroider-build/embroider/pull/1321 <@void_mAlex>
+  - is this essentially done?
+  - A: yes... mostly... there are some reviews that still need to happen
+  - A: (some details about specific commits that need extra review)
+  - A: should we merge it with any skipped tests?
+  - EF: we can land it with some skipped tests since it's so much better
+  - EF: as long as we're not submitting any tests that are passing with wrong semantics
+  - if there are we can make them fail and then skip them and come back to them
+  - A: I would love to dig more into one of the tests if we have time today
+  - EF: might get back to it if we have time
+
+<10 min>  last weeks topics
+- thoughts/decision on https://github.com/embroider-build/embroider/issues/1330 <@simon>
+
+  - SI: this all depends on the lock step strategy we have for embroider
+  - (discussion about the benefits of this approach)
+  - EF: we could just implement this using the existing ember-source-channel-url
+  - this would prevent more infra
+  - SI: maybe we merge the carrat PR
+  - EF: yes! merged
+  - we should try to implement the the source channel thing upstream. Anyone who wants to take it
+
+<10-15min> empty @ember/test-helpers dependency in the embroider output <@mansona>
```

Comment:
Started some prep:
 - https://github.com/emberjs/ember-test-helpers/pull/1324
 - https://github.com/emberjs/ember-test-helpers/pull/1325
 
 Previous attempt: 
  - https://github.com/emberjs/ember-test-helpers/pull/1249
     - abandoned due to lack of engagement